src/libraries/Common/src/System/Net/Http/Http3/Frames/Http3ErrorCode.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if KESTREL
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http3
```

Comment:
Yeah go ahead and make the changes here, we will react in Kestrel to the namespace changes.

Reply:
> Only IHttpHeadersHandler needed to preserve the namespace because it's public.

We're adding new methods to the interface.  That's similarly breaking, no?  I thought the purpose of the pubternals approach was that API changes were permitted.

Reply:
I think we can change IHttpHeadersHandler to be in the System.Net.Http namespace, but IIRC we use IHttpHeadersHandler in our benchmarks repo. cc @sebastienros we may update a namespace that we'd need to react to in benchmarks, that okay?

Reply:
Always ok if you apply the change in the benchmarks too ;)

Reply:
IHttpHeadersHandler is an internal type in System.Net.Http.dll, it doesn't matter what the namespace is there.

IHttpHeadersHandler is a public (pubternal) type in Kestrel. Yes we can make API changes to pubternal types, that's not the question here. We should not use the System.Net.Http namespace for public types in Kestrel.